Add documention for the ACLK (#8413)
netdata/netdata: claim/README.md

> Claiming can be performed through a SOCKS5 proxy. To do this when calling the script directly supply the proxy

Reviewer: We can also use http proxies

Reviewer: Additionally for SOCKS5 support `curl` must be present in system as `wget` doesn't support SOCKS5 unfortunatelly. This is however handled in claiming script with error message. (it is more of an issue if claiming from agent as that message then goes to error.log and is not instantly visible)

endpoint as:

```bash
netdata-claim.sh -token=MYTOKEN1234567 -rooms=room1,room2 -proxy=socks5h://127.0.0.1:11081
```

> When claiming via the `netdata` binary set the following options in the config:

Reviewer 2: Why does claiming via the binary require a socks5 proxy but the script does not - this doesn't make sense.

Author: Can you explain your concern a bit more? I'm not sure I follow. The command shows explicitly setting a proxy via the `-proxy=` argument.

Reviewer 2: The text has changed a bit and I can't see the problem. I think it was the context of the sentence but it seems to work ok now.

Author: Cool. Thanks for taking another peek.

> When claiming via the `netdata` binary, set the following options in your `netdata.conf` file:

Reviewer: `When claiming via the `netdata` binary, set the following options in your `netdata.conf` file`
add
`in case you need to use proxy to connect to internet:` ?

```conf
[cloud]
    proxy = socks5://X.X.X.X:YYYY
```

Proceed to claim using the command-line syntax:

```conf
/usr/sbin/netdata -D -W "claim -token=MYTOKEN1234567 -rooms=room1,room2"
```

> > Please note: if you supply the proxy endpoint in the configuration then it will also be used to tunnel the agent cloud

Reviewer 3: ```suggestion
> Please note: if you supply the proxy endpoint in the configuration, it is also used to tunnel the agent cloud
```

Reviewer: Agent to Cloud connection? Not sure what to understand under "agent cloud"

> link as well.

## Unclaim (remove) an agent from Netdata Cloud

The best method to remove an agent from Netdata Cloud is to unclaim it by deleting the `claim.d/` directory in your
Netdata configuration directory.

```bash
cd /etc/netdata   # Replace with your Netdata configuration directory, if not /etc/netdata/
rm -rf claim.d/
```

> You may need to use `sudo` or another method of elevating your privileges.

> Once you delete the `claim.d/` directory, the ACLK disconnects and Netdata Cloud removes the node from your account.

Reviewer 2: I don't think this is true - is there any functionality in the cloud that does this?

Reviewer: I think disconnection is not happening. It will just not connect again/next time.
